jfrog_platform/README.md

```diff
@@ -0,0 +1,81 @@
+
+
+## Overview
+The following describes how to configure Datadog to gather metrics from JFrog Artifactory and Xray.
```

Comment:
```suggestion

The following describes how to configure Datadog to gather metrics from JFrog Artifactory and AWS X-Ray.
```

Reply:
This tile does not involve AWS X-Ray. JFrog has a security product Xray.
I will change it on my system to "JFrog Xray" and do a separate commit.